.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="JavaScriptLibraryMappings">
-    <file url="PROJECT" libraries="{decap-cms}" />
-  </component>
-</project>
```

Comment:
No idea why this disappeared but I think it's not needed in first place

Reply:
It appeared when introducing Decap CMS in it's `index.html` but we barely touch that file so